src/Sequence.php

```diff
@@ -222,7 +222,7 @@ public function getGaps(): self
     }
 
     /**
-     * Returns the sum of all instances duration as expressed in seconds.
+     * Returns the sum of all instances durations as expressed in seconds.
      */
     public function getTotaTimestampInterval(): float
```

Comment:
why not simply call this method `Sequence::getInnerTimestampInterval` 🤔  
To mecurrently I think this method is a alias of  `Sequence->boundaries()->getTimestampInterval()`

Reply:
I don't get the comparison with `Sequence->boundaries()->getTimestampInterval()` but the name change of the method seems fine to me, changing it right now.

Reply:
Oh I get it now.
`Sequence->boundaries()->getTimestampInterval()` is not correct because uses the first and last datetime of the entire period.
This function takes all possible gaps in consideration and calculates only the actual duration of all periods inside the sequence.
Now I get the reason of the name change but it's not correct for what the function does.